### PR TITLE
Milestone 5: Adding Interaction

### PR DESCRIPTION
**Pull request by flpfar**
This PR adds the following to the project:

- Handle state in App component
- Functions handleClick to change the App state through Button
- Display calculations in the interface

Live version at https://calculator-magicians.herokuapp.com/

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+      } else {
+        result = `${total}${operation}`;
+      }
+    } else {
+      result = operation ? `${next}${operation}` : next;
+    }
+
+    return (
+      <div className="calculator">
+        <Display result={result} />
```
Comment (whiz25) [inline src/components/App.js]: - Pass `total` or `next` value. You may do something like:
```javascript
<Display value={this.state.next || this.state.total || '0'} />
```

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray: 
You have done a splendid job in this PR. Most requirements have been fulfilled. Good job. We however, need to make one required change before we can merge this PR. See my comment and suggestion.


### Martin Afani

File: src/components/App.js (hunk 1 above)
Comment (flpfar) [reply src/components/App.js]: Thanks for the suggestion. Done :heavy_check_mark:

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    case '+':
+      return n1.plus(n2);
+    case '-':
+      return n1.minus(n2);
+    case 'x':
+      return n1.times(n2).toString();
+    case '÷':
+      if (numberTwo === '0') {
+        return null;
+      }
```
Comment (shloch) [inline src/logic/operate.js:17]: return `INFINITY`  or `NaN`

**Review by shloch — APPROVED** [review]
Hello @flpfar 

calc

Good job.....works perfecty
However, take note of the small modification below before you merge
You get thumbs up

HAPPY CODING

**Pull request merged**